### PR TITLE
update ClientNetworkTransform information

### PR DESCRIPTION
**Pull request by NoelStephensUnity**
**Select the type of change:**

- [X] Small Changes - Typos, formatting, slight revisions
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
This just provides users with a bit more information about setting NetworkTransform into owner/client authoritative mode,

**Issue Number:** 
Based on feedback from the [SDK Github Issue 2189](https://github.com/Unity-Technologies/com.unity.netcode.gameobjects/issues/2189)

### CODE REVIEW

File: docs/components/networktransform.md [hunk 1]
```diff
@@ -51,8 +51,33 @@ darkImageSrc="/img/BufferedTick_Dark.png?text=DarkMode"/>
...
 
-Or you can directly add this line to your `manifest.json` file:
+You can use the existing `ClientNetworkTransform` that lives inside the Multiplayer Samples Utilities package. You can add this package via the `Package Manager` window in the Unity Editor by selecting `add from Git URL` and adding the following URL: `https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main`.
 
+Optionally, you can directly add this line to your `manifest.json` file:
 `"com.unity.multiplayer.samples.coop": "https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main"`
+
+You can also just create your own `ClientNetworkTransform` by deriving from `NetworkTransform`, overriding the `OnIsServerAuthoritative` virtual method, and return false:
```
[the remark this answers is not in the document]
Comment (NoelStephensUnity) [reply docs/components/networktransform.md:60]: Well... I figure we aren't going to be making many more changes to NetworkTransform at this point and the user's issue was that they had an old version of the ClientNetworkTransform.  When looking for information about "why" in our documentation there was nothing that the user saw as being different (i.e. go download the ClientNetworkTransform here).  _Some users don't create clones and just simply download the branch as a zip, grab what they want, and migrate that into their project._

Since we have no API documentation for NetworkTransform (I was going to point the user to the NetworkTransform API link and then realized there wasn't one), I figured just posting the few lines of code it takes to place a NetworkTransform into owner authoritative mode would be much easier to "spot the things that changed" than for them to re-download what they thought they had already downloaded.

[the remark this answers is not in the document]
Comment (armstrongl) [reply docs/components/networktransform.md:60]: I agree with @SamuelBellomo; in-doc samples are difficult to maintain and can easily become outdated. However, it sounds like this could benefit users.

That said, if we do include this excerpt, I suggest rewording it slightly:
> You can also create your own `ClientNetworkTransform` by deriving from `NetworkTransform`, overriding the `OnIsServerAuthoritative` virtual method, and returning false:

[the remark this answers is not in the document]
Comment (NoelStephensUnity) [reply docs/components/networktransform.md]: Ok, both are good suggestions and so I have adjusted the text copy and pointed the example directly to the Packages/com.unity.multiplayer.samples.coop/Utilities/Net/ClientAuthority/ClientNetworkTransform.cs file as opposed to the less manageable embedded code example.

File: docs/components/networktransform.md [hunk 2]
```diff
@@ -51,8 +51,14 @@ darkImageSrc="/img/BufferedTick_Dark.png?text=DarkMode"/>
 
 `NetworkTransform` always synchronizes positions from the server to the clients and position changes on the clients are not allowed. Netcode for GameObjects comes with a sample containing a `ClientNetworkTransform`. This transform synchronizes the position of the owner client to the server and all other client allowing for client authoritative gameplay.
 
-The `ClientNetworkTransform` lives inside the Multiplayer Samples Utilities package. You can add this package via the `Package Manager` window in the Unity Editor by selecting `add from Git URL` and adding the following URL: `https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main`
 
-Or you can directly add this line to your `manifest.json` file:
+You can use the existing `ClientNetworkTransform` that lives inside the Multiplayer Samples Utilities package. You can add this package via the `Package Manager` window in the Unity Editor by selecting `add from Git URL` and adding the following URL: `https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main`.
```
Comment (chrispope) [inline docs/components/networktransform.md:55]: Is Git required to be installed prior to attempting to add the package via from Git URL?

Comment (NoelStephensUnity) [reply docs/components/networktransform.md:55]: [It doesn't say you need Git installed ](https://docs.unity3d.com/Manual/upm-ui-giturl.html) in order to install a package from a Git URL. This might "just work" with the package manager.

Comment (armstrongl) [reply docs/components/networktransform.md:55]: Could we reword this a little bit:
> You can use the existing `ClientNetworkTransform` in the Multiplayer Samples Utilities package. 
> To add the Multiplayer Samples Utilities package:
> 1. Open the Package Manager by selecting **Window** > **Package Manager**.
> 2. Select the **plus button (+)** > **Add from git URL...**.
> 3. Copy and paste the following Git URL: `https://github.com/Unity-> Technologies/com.unity.multiplayer.samples.coop.git?path=/Packages/com.unity.multiplayer.samples.coop#main`
> 4. Select **Add**.

Comment (NoelStephensUnity) [reply docs/components/networktransform.md]: @armstrongl 
Done!

**Review by armstrongl — APPROVED** [review]

**Review by SamuelBellomo — APPROVED** [review]
